pkg/registry/networking/ingress/storage/storage_test.go

```diff
@@ -103,9 +115,8 @@ func newIngress(pathMap map[string]string) *networking.Ingress {
 			Namespace: namespace,
 		},
 		Spec: networking.IngressSpec{
-			Backend: &networking.IngressBackend{
-				ServiceName: defaultBackendName,
-				ServicePort: defaultBackendPort,
+			DefaultBackend: &networking.IngressBackend{
+				Service: serviceBackend,
```

Comment:
nit: whenever you use this, do `serviceBackend.DeepCopy()` to make sure we never pass this somewhere that mutates the original